server/index.js

```diff
@@ -2,7 +2,7 @@ import os from "os";
 import cors from 'cors'
 import express from "express";
 
-const port = 5000;
+const port = 4000;
```

Comment:
Is this to avoid AirPlay collisions in the future? As long as controls mirrors this, I think it's a smart move.